cmd/get-measurements/main.go

```diff
@@ -84,10 +94,21 @@ func runClient(cCtx *cli.Context) (err error) {
 		Version: common.Version,
 	})
 
+	// Sanity-check addr
 	if !strings.HasPrefix(addr, "https://") {
 		return errors.New("address needs to start with https://")
 	}
 
+	// Load expected measurements from file or URL (if provided)
+	var expectedMeasurements *multimeasurements.MultiMeasurements
+	if expectedMeasurementsPath != "" {
+		log.Info("Loading expected measurements from " + expectedMeasurementsPath + " ...")
+		expectedMeasurements, err = multimeasurements.New(expectedMeasurementsPath)
+		if err != nil {
+			return err
```

Comment:
maybe add a log.Error message here for better understanding what went wrong while debugging.

Reply:
an error returned in the main function is still printed to the console. but still a valid point 👍